### PR TITLE
Appending metadata instead of overriding.

### PR DESCRIPTION
**Pull request by leesiongchan**
There is a case when you want to append the same metadatas.

For example, 'og:image', we could provide several of 'og:image' metadatas for sharer to crawl.

### CODE REVIEW

**Comment by OskarStark** [comment]
Adding methods to an interface is BC break

**Comment by core23** [comment]
Please rebase this @ahlee2326 
Maybe you can create a seperate PR for the interface stuff, so we can merge this without a BC break.

**Comment by core23** [comment]
Ping @leesiongchan

**Comment by leesiongchan** [comment]
Sorry, it's been a long time, probably I will take some time to work on it by weekend.

**Comment by OskarStark** [comment]
did you do the work? @leesiongchan ?

File: Seo/SeoPageInterface.php [hunk 1]
```diff
@@ -44,6 +44,24 @@ public function addMeta($type, $name, $value, array $extras = array());
     /**
      * @param string $type
      * @param string $name
+     * @param string $value
+     * @param array  $extras
+     *
+     * @return mixed
+     */
+    public function setMeta($type, $name, $value, array $extras = array());
```
Comment (Soullivaneuh) [inline Seo/SeoPageInterface.php]: Should be commented to have a BC layer.

**Comment by Soullivaneuh** [comment]
According to the new [Sonata version management](https://github.com/sonata-project/SonataAdminBundle/issues/3053) and next major [release plan](https://github.com/sonata-project/SonataAdminBundle/issues/3731), this project has been refactored regarding branching and versioning.

If you see this message, your PR concerns a patch or a minor release and is not targeting the right branch.

So I'm closing this one, but don't see it as a refusal. If you think your work is still relevant and want to continue, feel free to reopen it on the right branch (e.g. the default one).

Regards.

**Pull request closed without merging**